### PR TITLE
add likes counter for each meal item

### LINKED ISSUE #32: [4pt]  - Display number of likes for each item on the Homepage - Student A
When the page loads, the webapp the [Involvement API](https://www.notion.so/microverse/Involvement-API-869e60b5ad104603aa6db59e08150270) to show the item likes and combines them with the data from the base API.

Remember that your page should make only 2 requests:
- one to the base API
- and **one** to the Involvement API.

This task **does not** include displaying the likes button (heart icon on the wireframe) for each item.

### PR DESCRIPTION
**Pull request by fickryiman**

### CODE REVIEW

File: src/components/homepage.js [hunk 1]
```diff
@@ -63,6 +74,22 @@ const homepageComponent = () => {
...
+      .then((response) => response.json())
+      .then((data) => data);
+
+    const likesCounter = cards.querySelectorAll('.likes-counter');
+    likesCounter.forEach((likeCounter) => {
+      const likesId = likeCounter.getAttribute('data-id');
+      const element = likeCounter;
+      likesAPI.forEach((likeAPI) => {
+        if (likesId === likeAPI.item_id) {
+          element.innerHTML = likeAPI.likes;
```
Comment (mahabubx7) [inline src/components/homepage.js:89]: Sorry! This cannot be incomplete.
> The likes are supposed to be visible and should update the counter as API returns.

**Review by mahabubx7 — CHANGES_REQUESTED** [review]
### :recycle: Status: Changes required! :recycle: 
You need to make sure that the like API is returning update like counts and also the DOM is updating with that value. It seems like counter is not working.

**Comment by fickryiman** [comment]
do you have the likes counter on your localstorage?
it is random, the DOM wouldn't;t display it if you don't have the likes counter data.
image

**Comment by fickryiman** [comment]
> ### ♻️ Status: Changes required! ♻️
> You need to make sure that the like API is returning update like counts and also the DOM is updating with that value. It seems like counter is not working.

do you mean to post the like counter? no, this is a feature to show and get the likes counter. for posting the like counter have their own kanban card.

**Review by mahabubx7 — APPROVED** [review]
Okay! seems to be fine :heavy_check_mark:

**Review by fickryiman — COMMENTED** [review]
likes counter functionality is working.

**Pull request merged**